chart/templates/_helpers.yaml

```diff
@@ -782,3 +782,83 @@ Where `.` is the global variables scope and `.Values.workers` the local variable
     {{- end -}}
   {{- end -}}
 {{- end -}}
+{{- define "workers_extra_envs" -}}
```

Comment:
You shouldn't need to do a helper per component, just pass the right map in from the template and have a single helper.

Reply:
Done